controllers/component_controller_test.go

```diff
@@ -1034,7 +914,7 @@ var _ = Describe("Component controller", func() {
 			createdHasComp := &appstudiov1alpha1.Component{}
 			Eventually(func() bool {
 				k8sClient.Get(context.Background(), hasCompLookupKey, createdHasComp)
-				return len(createdHasComp.Status.Conditions) > 0 && createdHasComp.GetLabels()["appstudio.has/component"] != ""
+				return len(createdHasComp.Status.Conditions) > 0 //&& createdHasComp.GetLabels()["appstudio.has/component"] != ""
```

Comment:
should we remove comment?

Reply:
Hard to explain - will address in a follow up PR